README.md

```diff
@@ -27,6 +27,26 @@ jobs:
       uses: icrawl/action-tsc@v1
 ```
 
+### Adding `tsc`
```

Comment:
While this may be helpful, this should already happen anyway, I don't necessarily want to teach people that they should add their devDeps to their devDeps.

I take the risk and maybe have to answer that in issues in the future.
So removing that point would be nice.

The latter is fine though!

Reply:
@iCrawl For some users it's not a dev-dependency - their tooling (for instance Visual Studio) has native TypeScript support. If you add tsc as a dev-dependency it can cause issues for VS (you can get a kind of duelling tsc versions, and it's mainly VS's fault) and it took me a while to figure that out. 

Not everyone uses npm dev dependencies (even if we use npm for application dependencies), especially .NET/Azure devs with Microsoft tools.

In any case, it would have been useful for me trying to figure this out.

Should I remove it? Or maybe it belongs in a different section?

Reply:
> For some users it's not a dev-dependency - their tooling (for instance Visual Studio) has native TypeScript support. 

Yes I understand that, but that is simply *wrong*.
Why you might ask? Well because the moment someone else works on your project too you need to make sure they also have a global dependency of it (if its not a devDep)

Hence saying its not necessarily my job to teach people how this works and I take full responsibility in having issues opened because of this if it should ever arise to be a problem for someone.

Reply:
I don't mean it's _not_ a dependency, it's just not an _npm_ dependency, and so not managed by the package.json list restored by yarn (even though you can get tsc from npm) - see also NuGet, Cargo Crates, or any other package manager. I'm totally with you on ensuring that the whole team can build the project, just not that _only_ being possible through npm.

But fair enough, I don't want to create issues for you. I've removed the section.